res/Sectors/M1105/centrax.txt

```diff
@@ -0,0 +1,460 @@
+# Generated by https://travellermap.com
+# 2018-11-05T13:05:23+00:00
+
+# Centrax
+# 4,3
+
+# Name: Centrax (hv)
+
+# Milieu: M1105
+
+# Credits: Centrax sector was designed by Mark Humphreys. After dot-map by Leroy W. L. Guatney http://home.earthlink.net/~ngc5139/FDA/
+
+# Author:    Mark Humphreys
+
+# Subsector A: Kornaskereq
+# Subsector B: Slomua
+# Subsector C: Wrunoothon
+# Subsector D: Liana
+# Subsector E: Neerndantena
+# Subsector F: Konpeq
+# Subsector G: Wrilootanskuk
+# Subsector H: Aexun
+# Subsector I: Blexiontcar
+# Subsector J: Arephlaft
+# Subsector K: Uciennatin
+# Subsector L: Wruoruaft
+# Subsector M: Meexiannelizi
+# Subsector N: Araurnuansk
+# Subsector O: Pila
+# Subsector P: Wrogavaoqgior
+
+# Alleg: CsHv: "Hiver Client State"
+# Alleg: CaIa: "Ithklur Client State"
```

Comment:
The "CaIa" code isn't used, nor is the  more expected "CsIa".

Reply:
All of these comment lines in .txt files (subsector names, allegiances, etc) are completely ignored. It would be better to just remove them.

Reply:
I have a use for them, as I need them there as part of the analysis. This mimics the T5 Sector Data format. But if they're not necessary for your workings to update them I'll stop making suggestions to fix them.

Reply:
I missed that one. I'll commit this (and change CsHv to "Client state, Hive Federation", so it matches the entry in allegience_codes.tab.

As to removing the whole lot, I'll wait until consensus it reached! As a note I think the API to get a Sector puts them in, even if they are not in the actual file in the project.